src/views/Settings.vue

```diff
@@ -198,6 +247,50 @@ export default {
       this.currentPassword = "";
       this.newPassword = "";
       this.confirmNewPassword = "";
+    },
+    async shutdownPrompt() {
+      // Get user consent first
+      const approved = await this.$bvModal.msgBoxConfirm(
+        'Your Umbrel will be unresponsive untill you can get physical access to the device to power it back on.',
+        {title: "Are you sure?"}
+      );
+      if (!approved) {
+        return;
+      }
+
+      // TODO: Send shutdown request
+
+      this.$bvToast.toast(
+        `System is shutting down`,
+        {
+          title: "Your Umbrel will become unresponsive soon",
+          autoHideDelay: 3000,
+          variant: "danger",
```

Comment:
We can change the variant to `warning` (for yellow background color), as `danger` is mostly used for errors, etc.